pysatModels/models/ucar_tiegcm.py

```diff
@@ -112,9 +122,20 @@ def init(self):
 #
 # Use local and default pysat methods
 
-# Set the list_files routine
-fname = 'tiegcm_icon_merg2.0_totTgcm.s_{day:03d}_{year:4d}.nc'
-supported_tags = {'': {'': fname}}
+# Set the list_files routine. CDAWeb files have an intermediate day directory.
+# Accounted for with the leading '*'.
+icon_fname = ''.join(['ICON_L4-3_TIEGCM_{year:04d}-{month:02d}-{day:02d}_',
+                      'v{version:02d}r{revision:03d}.NC'])
+icon_fname = os.path.join('*', icon_fname)
+
+# Files from UCAR
+ucar_fname = 'tiegcm_icon_merg2.0_totTgcm.s_{day:03d}_{year:4d}.nc'
```

Comment:
These could live as a dict with tags as keys, making the `supported_tags` definition simpler.  Or just define them within `supported_tags`.